client/src/main/java/io/pravega/client/segment/impl/SegmentMetadataClient.java

```diff
@@ -49,10 +49,9 @@
      * Deletes all data before the offset of the provided segment.
      * This data will no longer be readable. Existing offsets are not affected by this operations. 
      * The new startingOffset will be reflected in {@link SegmentMetadataClient#getSegmentInfo(String).startingOffset}.
-     * @param segment The segment to truncate.
```

Comment:
The javadoc link to getSegmentInfo has an incorrect input parameter.

Reply:
Fixed